env.daint.sh

```diff
@@ -201,17 +197,16 @@ setFortranEnvironment()
     fi
 
     # standard modules (part 1)
-    module load cmake
+    module load CMake
     if [ "${target}" == "gpu" ] ; then
-        module load craype-accel-nvidia35
-        module swap cudatoolkit/7.0.28-1.0502.10742.5.1
+        module load craype-accel-nvidia60
     fi
 
     # compiler specific modules
     case "${compiler}" in
     cray )
-        module unload cce
-        module load cce/8.4.0
+#        module unload cce
```

Comment:
Is it not possible to keep a fixed cray cce version, if we don't load explicitly we will be using the default which may change wihtout explicit notice